git/pre-commit

```diff
@@ -2,6 +2,16 @@
 
 if [ "$NOLINT" ]; then exit 0; fi
 
+if [ ! $TRAVIS = true ]
```

Comment:
I'm comparing boolean values as suggested here: http://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script
And negating the expression as explained here: https://bash.cyberciti.biz/guide/Logical_Not_!